src/main/java/edu/cornell/kfs/coa/businessobject/AccountReversionGlobal.java

```diff
@@ -272,7 +274,19 @@ public void setCashReversionFinancialChartOfAccountsCode(String cashReversionFin
         this.cashReversionFinancialChartOfAccountsCode = cashReversionFinancialChartOfAccountsCode;
     }
 
-    /**
+    public Boolean isReversionActiveIndicator() {
```

Comment:
I think getReversionActiveIndicator is the only one strictly needed since the property is a Boolean object and not a boolean primitive. There is another property on the same class that also has this duplication, though, so I added both for consistency with that property. If you think it's too redundant, I can remove the extra method.